dev/PipsPager/PipsPager.cpp

```diff
@@ -516,6 +477,93 @@ void PipsPager::OnNextButtonClicked(const IInspectable& sender, const winrt::Rou
     SelectedPageIndex(SelectedPageIndex() + 1);
 }
 
+void PipsPager::OnGotFocus(const winrt::RoutedEventArgs& args)
+{
+    if (const auto btn = args.OriginalSource().try_as<winrt::Button>())
+    {
+        // If the element inside the Pager is already keyboard focused
+        // and the user will use the mouse to focus on something else
+        // the LostFocus will not be triggered on keyboard focused element
+        // while GotFocus will be triggered on the new mouse focused element.
+        // We account for this scenario and update m_isFocused in case
+        // user will use mouse while being in keyboard focus.
+        if (btn.FocusState() != winrt::FocusState::Pointer)
+        {
+            m_isFocused = true;
+            UpdateNavigationButtonVisualStates();
+        }
+        else
+        {
+            m_isFocused = false;
```

Comment:
Yes, because the Focus Rect will disappear (due to the change of kind of focus) and when the user will move the mouse away - the buttons should disappear.

Reply:
Ah I see.